examples/ALOS2437590500-220630_WWDR2.2GUA/ALOS2437590500-220630_WWDR2.2GUA.json

```diff
@@ -0,0 +1,182 @@
+{
+  "type": "Feature",
+  "stac_version": "1.0.0",
+  "id": "ALOS2437590500-220630_WWDR2.2GUA",
+  "properties": {
+    "proj:epsg": 32651,
+    "proj:bbox": [
+      374612.5,
+      2689112.5,
+      780462.5,
+      3087012.5
+    ],
+    "proj:shape": [
+      16234,
+      15916
+    ],
+    "sar:frequency_band": "L",
+    "sar:center_frequency": 1.2575,
+    "sar:observation_direction": "right",
+    "sar:instrument_mode": "WWD",
+    "sar:polarizations": [
+      "HH",
+      "HV"
+    ],
+    "sar:product_type": "GRD",
+    "sat:orbit_state": "ascending",
+    "datetime": "2022-06-30T15:58:28.260000Z"
+  },
+  "geometry": {
+    "type": "Polygon",
+    "coordinates": [
+      [
+        [
+          121.732262,
+          27.339137
+        ],
+        [
+          122.31674,
+          24.313445
+        ],
+        [
+          125.775968,
+          24.879538
+        ],
+        [
+          125.285874,
+          27.889129
+        ],
+        [
+          121.732262,
+          27.339137
+        ]
+      ]
+    ]
+  },
+  "links": [
+    {
+      "rel": "root",
+      "href": "../collection.json",
+      "type": "application/json",
+      "title": "PALSAR-2 ScanSAR NRB L2.2"
+    },
+    {
+      "rel": "collection",
+      "href": "../collection.json",
+      "type": "application/json",
+      "title": "PALSAR-2 ScanSAR NRB L2.2"
+    },
+    {
+      "rel": "parent",
+      "href": "../collection.json",
+      "type": "application/json",
+      "title": "PALSAR-2 ScanSAR NRB L2.2"
+    }
+  ],
+  "assets": {
+    "HH_SLP": {
+      "href": "ALOS2437590500-220630_WWDR2.2GUA_HH_SLP.tif",
```

Comment:
Does this file live in `tests/data-files`? If so, it'd be good to fix up this href to point to the file.

Reply:
Ah, for some reason I thought I was supposed to drop that. Fixed.